config-templates/ibm/ibm-system-storage-block-csi-driver/1.11.1/custom-parameters.json

```diff
@@ -6,5 +6,38 @@
     "default": "default",
     "required": "true",
     "category": "config"
+  },
+  {
+    "description": "The name of the secret to create.",
+    "displayname": "Secret Name",
+    "name": "secret-name",
+    "required": "true",
+    "regex": "^([a-z0-9]|[a-z0-9][a-z0-9.-]*[a-z0-9]+)$",
+    "min-length": 1,
+    "max-length": 64,
+    "category": "config"
+  },
+  {
+    "description": "The address of the management server.",
```

Comment:
@ArbelNathan do you have an example of how this address should look? or a command to retreive or links for how to find it?

Reply:
this is the storage address, could be URL or IP. (i.e. exanple-cluster.xiv.ibm.com)